posthog-core/src/index.ts

```diff
@@ -1252,6 +1258,14 @@ export abstract class PostHogCore extends PostHogCoreStateless {
       this.capture('$feature_flag_called', {
         $feature_flag: key,
         $feature_flag_response: response,
+        $feature_flag_bootstrapped_response: this.getPersistedProperty<PostHogDecideResponse['featureFlags']>(
+          PostHogPersistedProperty.BootstrapFeatureFlags
+        )?.[key],
+        $feature_flag_bootstrapped_payload: this.getPersistedProperty<PostHogDecideResponse['featureFlagPayloads']>(
+          PostHogPersistedProperty.BootstrapFeatureFlagPayloads
+        )?.[key],
+        // If we haven't yet received a response from the /decide endpoint, we must have used the bootstrapped value
+        $used_bootstrap_value: !this.getPersistedProperty(PostHogPersistedProperty.DecideEndpointWasHit),
```

Comment:
I think it's ok having this but you could also just compare the flag (the cached flag - likely not there if decide isn't called, and the value from the bootstrap flags), so you know if it's being used from the bootstrap or not, right?
Since using storage is not super fast, the less we need the better, your call here, but if possible I'd evaluate this at runtime.

Reply:
Are you referring to `PostHogPersistedProperty.FeatureFlags` or a platform specific cache?

I was hoping we could avoid this as well, but we store bootstrapped flags in `PostHogPersistedProperty.FeatureFlags` upon initialization, so it would already be there